.pre-commit-config.yaml

```diff
@@ -14,6 +14,7 @@ repos:
           - huxtable
           - r2rtf
           - vctrs
+          - checkmate
```

Comment:
I believe that I introduced it as it is very useful to check inputs etc... but I was thinking we should discuss it. It is used only in `toString` and imported for all of its functions, a bit too much probably. Still is imported by all other packages so it is not a game changer from the user's perspective imo